<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="localhost.testing.MultiInputs" tests="6" time="1.252" ignored="0" failures="0" timestamp="29 Nov 2017 22:32:49 GMT" skipped="0" hostname="DESKTOP-0JEVNSK" errors="1">
  <testcase name="addWetDryContacts" time="1.000" classname="localhost.testing.MultiInputs"/>
  <testcase name="clearInputs" time="0.001" classname="localhost.testing.MultiInputs"/>
  <testcase name="clearGroupAll" time="0.000" classname="localhost.testing.MultiInputs"/>
  <testcase name="closeBrowser" time="0.000" classname="localhost.testing.MultiInputs">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at localhost.testing.MultiInputs.closeBrowser(MultiInputs.java:195)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.TestRunner.afterRun(TestRunner.java:983)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
    </error>
  </testcase> <!-- closeBrowser -->
  <testcase name="createGroupAll" time="0.000" classname="localhost.testing.MultiInputs"/>
  <testcase name="checkInputs" time="0.251" classname="localhost.testing.MultiInputs"/>
</testsuite> <!-- localhost.testing.MultiInputs -->
